extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -168,6 +180,15 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
      * to an amp-ad-adsense element. Thus, if we are an amp-ad, we can be sure
      * that it has been verified.
      */
+    if (this.isResponsive_() && !this.hasValidResponsiveHeight_()) {
```

Comment:
This looks good, just one minor change which is to remove the hasValidResponsiveHeight_ function entirely as splitting it off doesn't add much IMO:

if (this.isResponsive_() && this.element.getAttribute('height') != ADSENSE_RSPV_WHITELISTED_HEIGHT) { ... }